### PR TITLE
feat:created bottom sheet for multimeter

### LINKED ISSUE #1074: convert the dialog guide of multimeter into bottonsheet guide
**Actual Behaviour**

Currentlty the multimeter is having dialog guide, that can only be used once.

**Expected Behaviour**

Guide should be viewable anytime so it can be implemented using bottomsheet.

**Steps to reproduce it**

Multimeter Activity->convert the dialog into bottomsheet

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1074 

**Changes**:
Created bottom sheet for multimeter

**Screenshot/s for the changes**: 
whatsapp image 2018-06-18 at 17 41 34

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/2111151/app-debug.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal Conflicts resolved.

**Comment by Avjeet** [comment]
@CloudyPadmal yes #1050  needs to properly reviewed and merged so that same code can be applied to other instruments.

**Comment by abhinavraj23** [comment]
@CloudyPadmal @mariobehling Conflicts resolved

**Comment by CloudyPadmal** [comment]
@Avjeet @harsh-2711 @Vikum94 please review this one soon

File: app/src/main/res/layout/activity_multimeter.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (Avjeet) [inline app/src/main/res/layout/activity_multimeter.xml]: I dont think you need this namespace

**Pull request merged**